specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,35 @@
+# Pricing Unit
+
+The Pricing Unit represents a provider-specified measurement unit for determining unit prices, indicating how a provider rates measured usage and purchase quantities considering pricing rules like block pricing. Common examples include the number of hours for compute appliance runtime (e.g. `Hours`), gigabyte-hours for a storage appliance (e.g., `GB-Hours`), or an accumulated count of requests for a network appliance or API service (e.g., `1000 Requests`). Pricing Unit complements the [Pricing Quantity](#pricingquantity) metric. Distinct from the [Usage Unit](#usageunit), it focuses on pricing and cost, not resource and service consumption, often at a coarser granularity.
```

Comment:
This uses an example with a block size, but I thought the Unit Format doesn't not want numbers in the value, right? Should we remove that example?

Reply:
Is a quantifier dimension eg `pricingBlockSize` in 1.0 or post-1.0?

Reply:
Pricing Unit is logically the same as what providers have today with a recommendation to use consistent formatting specified in Unit Format. Block size can be a fast follow in v1.0+ since it's only necessary in a small number of (still important) use cases. Primary need is for providing a way for users to make sense of the rates/quantities/cost and that's accomplished by this column.

Reply:
One of the notes in the Discussion/Scratch space refers to some suggestions for 1.0+